src/infoPanel/infoPanel.c

```diff
@@ -112,29 +127,56 @@ int main(int argc, char *argv[])
 				strncpy(image_path, argv[++i], STR_MAX-1);
 			else if (strcmp(argv[i], "-j") == 0 || strcmp(argv[i], "--images-json") == 0)
 				strncpy(images_json_path, argv[++i], STR_MAX-1);
+			else if (strcmp(argv[i], "-d") == 0 || strcmp(argv[i], "--directory") == 0)
+				strncpy(images_dir_path, argv[++i], STR_MAX-1);
+			else if (strcmp(argv[i], "-s") == 0 || strcmp(argv[i], "--show-theme-controls") == 0)
+				show_theme_controls = true;
 			else if (strcmp(argv[i], "-a") == 0 || strcmp(argv[i], "--auto") == 0)
 				wait_confirm = false;
 		}
 	}
 
-	SDL_Init(SDL_INIT_VIDEO);
-	SDL_ShowCursor(SDL_DISABLE);
-	TTF_Init();
-	
-	SDL_Surface *video = SDL_SetVideoMode(640, 480, 32, SDL_HWSURFACE);
-	SDL_Surface *screen = SDL_CreateRGBSurface(SDL_HWSURFACE, 640, 480, 32, 0, 0, 0, 0);
+	signal(SIGINT, sigHandler);
+    signal(SIGTERM, sigHandler);
+
+	SDL_InitDefault(true);
+
+	lang_load();
```

Comment:
You need to call `settings_load` before this, unless you on purpose only want to load `en.lang`?